templates/base.html

```diff
@@ -11,6 +11,8 @@
 </head>
 <body {% block body_tag_attributes %}{% endblock %}>
 {% block content %}{% endblock content %}
+
+<script src="static/functions.js"></script>
```

Comment:
We don't want this Javascript loaded on every page (e.g., the pong game page), so it shouldn't be in base.html, rather index.html.